eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/UnitTests.hs

```diff
@@ -710,8 +710,8 @@ testProducedOverMaxWord64 =
       st =
         runShelleyBase $
           applySTSTest @(ShelleyLEDGER C) (TRC (ledgerEnv, ledgerState, tx))
-   in -- We test that the serialization of the predicate failure does not return bottom
-      serialize' shelleyProtVer st @?= serialize' shelleyProtVer st
+   in -- We test that the predicate failure does not return bottom
+      pure $! rnf st
```

Comment:
@JaredCorduan this is the proper way to check that the type does not return bottom :wink:

Reply:
much better, thanks!